t/01-format.t

```diff
@@ -0,0 +1,38 @@
+use strict;
+use warnings;
+
+use Test::More;
+use File::Basename;
+use File::Find ();
+use Locale::PO;
+
+chdir dirname(__FILE__) . "/..";
+
+use vars qw/*name *dir *prune/;
+*name   = *File::Find::name;
+*dir    = *File::Find::dir;
+*prune  = *File::Find::prune;
+
+sub wanted;
+
+# Since there are currently no plural or numeric translations,
+# this should suffice for now
```

Comment:
Curious, what would it look like if we *did* get some of those? Would CI immediately break or would it quietly stop working or similar?

Reply:
It would not work for those. I believe CI would pass, but we would miss:

- Numeric format specifiers, e.g. `%d`
- A pass over multiple msgstrs - for plurals they're a list `msgstr0, msgstr1 .. msgstrN`

I would probably start looking for a complete thirdparty verification solution when we got to that point, rather than attempting to build upon that function. I haven't yet assimilated the full set of gettext docs - perhaps it's something that exists.